find/main/routes.py

```diff
@@ -45,6 +48,7 @@ def index():
 
 @bp.route("/start", methods=["GET", "POST"])
 @login_required(return_app=SupportedApp.POST_AWARD_FRONTEND)
```

Comment:
Can we make a backlog task to move this into utils and have an optional argument to specify an email domain restriction?

Reply:
Have added the [backlog task here](https://dluhcdigital.atlassian.net/browse/FPASF-474)